integrations/tensorboard/Neptune-TensorBoard.ipynb

```diff
@@ -0,0 +1,765 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "Neptune gives you an option of logging data under a public folder as an anonymous user. This is great when you are just trying out the application and don't have a Neptune account yet.  \n",
+    "\n",
+    "---\n",
+    "\n",
+    "If you already have a [Neptune account](https://neptune.ai/register), you can create your own experiment and start logging to it using your personal API token. Pass your `username` to the `project_qualified_name` argument of the `neptune.init()` method: `project_qualified_name='YOUR_USERNAME/YOUR_PROJECT_NAME`. If you don't have a project yet, keep `/sandbox` at the end. The `sandbox` project is automatically created for you."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## Step 2: Create an experiment"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "neptune.create_experiment('tensorboard-logging')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## Step 3: Run ``neptune_tensorboard.integrate_with_tensorflow()``"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "import neptune_tensorboard\n",
+    "\n",
+    "neptune_tensorboard.integrate_with_tensorflow()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## Step 4: Add your training code"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "import tensorflow as tf\n",
+    "import datetime\n",
+    "\n",
+    "mnist = tf.keras.datasets.mnist\n",
+    "\n",
+    "(x_train, y_train),(x_test, y_test) = mnist.load_data()\n",
+    "x_train, x_test = x_train / 255.0, x_test / 255.0\n",
+    "\n",
+    "def create_model():\n",
+    "    return tf.keras.models.Sequential([\n",
+    "        tf.keras.layers.Flatten(input_shape=(28, 28)),\n",
+    "        tf.keras.layers.Dense(512, activation='relu'),\n",
+    "        tf.keras.layers.Dropout(0.2),\n",
+    "        tf.keras.layers.Dense(10, activation='softmax')\n",
+    "      ])\n",
+    "\n",
+    "model = create_model()\n",
+    "model.compile(optimizer='adam',\n",
+    "              loss='sparse_categorical_crossentropy',\n",
+    "              metrics=['accuracy'])\n",
+    "\n",
+    "log_dir = \"logs/fit/\" + datetime.datetime.now().strftime(\"%Y%m%d-%H%M%S\")\n",
+    "tensorboard_callback = tf.keras.callbacks.TensorBoard(log_dir=log_dir, histogram_freq=1)\n",
```

Comment:
Are you sure that you need to create TB callback for ``neptune_tensorboard.integrate_with_tensorflow()`` to work?

In the past I simply used integrate_with_tensorflow without TB.

Reply:
If you log it without TensorBoard callback nothing will get logged I think.
Example of how it looks:
https://ui.neptune.ai/shared/tensorboard-integration/e/IN-25/logs